GrandCentralBoard/Widgets/Image/ImageWidget.swift

```diff
@@ -9,18 +9,25 @@ import GCBCore
 
 final class ImageWidget: WidgetControlling {
 
-    let widgetView: ImageWidgetView
+    private let widgetView: ImageWidgetView
+    private let widgetViewWrapper: UIView
+
     let sources: [UpdatingSource]
     let isHeaderVisible: Bool
 
     init(view: ImageWidgetView, sources: [UpdatingSource], isHeaderVisible: Bool) {
         self.widgetView = view
         self.sources = sources
         self.isHeaderVisible = isHeaderVisible
+
+        let viewModel = WidgetTemplateViewModel(title: "CAT PHOTOS", subtitle: "NEWEST CAT PROFILES", contentView: widgetView)
+        let layoutSettings = WidgetTemplateLayoutSettings(contentMargin: UIEdgeInsetsZero, displayContentUnderHeader: true)
+        widgetViewWrapper = WidgetTemplateView.viewWithViewModel(viewModel, layoutSettings: layoutSettings)
```

Comment:
Sorry, I didn't see it yesterday in CR. If `isHeaderVisible` equals `false` we create `WidgetTemplateView`, but never use it. If we know that we don't need a header, let's not create it. What about:

```swift
private let widgetView: ImageWidgetView
private let mainView: UIView

// ...

if (isHeaderVisible) {
   let wrapper = ...
   widgetView = view
   mainView = wrapper
} else {
   widgetView = view
   mainView = view
}

// ...

var view: UIView {
   return mainView
}

```
?